src/server/controllers/movie.js

```diff
@@ -1,31 +1,44 @@
-const jwt = require('jsonwebtoken');
-const { PrismaClient } = require('@prisma/client');
-const prisma = new PrismaClient();
+const jwt = require('jsonwebtoken')
+const { PrismaClient } = require('@prisma/client')
+const prisma = new PrismaClient()
 
-const jwtSecret = 'mysecret';
+const jwtSecret = 'mysecret'
```

Comment:
you could use the **.env** to store these values ☕ Would be nicer in a real-world project